JS.md

```diff
@@ -166,6 +166,29 @@ Otherwise, this function performs synchronous compilation of the `BufferSource`:
   the validated `Ast.module`.
 * On failure, a new `WebAssembly.CompileError` is thrown.
 
+### `WebAssembly.Module.exports`
```

Comment:
Will it make sense to add "prototype" to denote object instance property (vs "static" property of Module): `WebAssembly.Module.prototype.exports`. Alternative is to start section "Properties of the WebAssembly.Module prototype" with subsection for every property.